Extensions/BitmapText/bitmaptextruntimeobject.js

```diff
@@ -0,0 +1,270 @@
+  this.onCreated();
+};
+
+gdjs.BitmapTextRuntimeObject.prototype = Object.create(
+  gdjs.RuntimeObject.prototype
+);
+gdjs.registerObject('BitmapText::BitmapText', gdjs.BitmapTextRuntimeObject);
+
+gdjs.BitmapTextRuntimeObject.prototype.getRendererObject = function () {
+  return this._renderer.getRendererObject();
+};
+
+/**
+ * @param {BitmapTextObjectDataType} oldObjectData
+ * @param {BitmapTextObjectDataType} newObjectData
+ */
+gdjs.BitmapTextRuntimeObject.prototype.updateFromObjectData = function (
+  oldObjectData,
+  newObjectData
+) {
+  if (oldObjectData.content.opacity !== newObjectData.content.opacity) {
+    this.setOpacity(newObjectData.content.opacity);
+  }
+  if (oldObjectData.content.text !== newObjectData.content.text) {
+    this.setText(newObjectData.content.text);
+  }
+  if (oldObjectData.content.fontColor !== newObjectData.content.fontColor) {
+    this._fontColor = newObjectData.content.fontColor;
+    this._renderer.updateColor();
+  }
+  if (
+    oldObjectData.content.fontResourceName !==
+    newObjectData.content.fontResourceName
+  ) {
+    this.setFontFamily(newObjectData.content.fontResourceName);
+  }
+  if (oldObjectData.content.fontSize !== newObjectData.content.fontSize) {
+    this.setFontSize(newObjectData.content.fontSize);
+  }
+  if (oldObjectData.content.wordWrap !== newObjectData.content.wordWrap) {
+    this.setWordWrap(newObjectData.content.wordWrap);
+  }
+  if (oldObjectData.content.align !== newObjectData.content.align) {
+    this.setAlignment(newObjectData.content.align);
+  }
+
+  return true;
+};
+
+/**
+ * Initialize the extra parameters that could be set for an instance.
+ * @private
+ */
+gdjs.BitmapTextRuntimeObject.prototype.extraInitializationFromInitialInstance = function (
+  initialInstanceData
+) {
+  if (initialInstanceData.customSize) {
+    this.setWrappingWidth(initialInstanceData.width);
+  }
+};
+
+gdjs.BitmapTextRuntimeObject.prototype.onDestroyFromScene = function (
+  runtimeScene
+) {
+  // TODO: mark the font as not used anymore in the "BitmapFont manager".
+
+  gdjs.RuntimeObject.prototype.onDestroyFromScene.call(this, runtimeScene);
+};
+
+/**
+ * Set the text to display.
+ */
+gdjs.BitmapTextRuntimeObject.prototype.setText = function (text) {
+  this._text = text;
+  this._renderer.updateTextContent();
+};
+
+/**
+ * Get the text displayed by the object.
+ */
+gdjs.BitmapTextRuntimeObject.prototype.getText = function () {
+  return this._text;
+};
+
+gdjs.BitmapTextRuntimeObject.prototype.setColor = function (rgbColorString) {
+  const splitValue = rgbColorString.split(';');
+  if (splitValue.length !== 3) return;
+  const hexColor =
+    '#' +
+    gdjs.rgbToHex(
+      parseInt(splitValue[0], 0),
+      parseInt(splitValue[1], 0),
+      parseInt(splitValue[2], 0)
+    );
+  this._fontColor = hexColor;
+  this._renderer.updateColor();
+};
+
+gdjs.BitmapTextRuntimeObject.prototype.getColor = function () {
```

Comment:
The `getColor` function needs to return an "R;G;B" string in order to not crash the tween extension.
See: #1956